engine/src/test/scala/cromwell/engine/workflow/SingleWorkflowRunnerActorSpec.scala

```diff
@@ -78,7 +79,7 @@ abstract class SingleWorkflowRunnerActorSpec extends CromwellTestkitSpec {
   def singleWorkflowActor(sampleWdl: SampleWdl = ThreeStep, managerActor: => ActorRef = workflowManagerActor(),
                           outputFile: => Option[Path] = None): Unit = {
     val actorRef = createRunnerActor(sampleWdl, managerActor, outputFile)
-    val futureResult = actorRef ? RunWorkflow
+    val futureResult = actorRef.?(RunWorkflow)(timeout = new Timeout(TimeoutDuration))
```

Comment:
This is the epitome of a `tol` but I say it because I always forget even when I'm writing tests. Scalatest 3.0 has all kinds of new support for async/future based tests. If some brave soul were to take a look it might help clean up some of our tests.

I even have a cheat sheet on my desk (by the face wall) if someone wants to steal it